spec/intl.html

```diff
@@ -1187,6 +1219,36 @@ <h1>Temporal.PlainDate.prototype.toLocaleString ( [ _locales_ [ , _options_ ] ]
             1. Return ? FormatDateTime(_dateFormat_, _temporalDate_).
           </emu-alg>
         </emu-clause>
+
+        <emu-clause id="sec-get-temporal.plaindate.prototype.era">
+          <h1>get Temporal.PlainDate.prototype.era</h1>
+          <p>
+            `Temporal.PlainDate.prototype.era` is an accessor property whose set accessor function is *undefined*.
+            Its get accessor function performs the following steps:
+          </p>
+          <emu-alg>
+            1. Let _plainDate_ be the *this* value.
+            1. Perform ? RequireInternalSlot(_plainDate_, [[InitializedTemporalDate]]).
+            1. Let _calendar_ be _plainDate_.[[Calendar]].
+            1. Let _method_ be ? Get(_calendar_, *"era"*).
+            1. Return ? Call(_method_, _calendar_, « _plainDate_ »).
```

Comment:
Any reason not to use Invoke here? (same with all the Get + Calls)

Reply:
I hadn't encountered Invoke before! ... Good suggestion, I'll open a new issue for that.